grr/server/grr_response_server/gui/ui/proxy.conf.json

```diff
@@ -1,10 +1,10 @@
 {
-  "/api": {
-    "target": "http://localhost:8000",
-    "secure": false
-  },
-  "/static": {
-    "target": "http://localhost:8000",
-    "secure": false
-  }
-}
+    "/api": {
```

Comment:
Google's styleguide suggests indentation with two spaces (not four). Please reformat to 2-space indentation.

Reply:
Solved!